SwrveConversationSDK/src/main/java/com/swrve/sdk/SwrveConversationHelper.java

```diff
@@ -0,0 +1,80 @@
+package com.swrve.sdk;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.drawable.Drawable;
+import android.graphics.drawable.LayerDrawable;
+import android.graphics.drawable.ShapeDrawable;
+import android.graphics.drawable.shapes.RoundRectShape;
+
+import com.swrve.sdk.conversations.R;
+
+public class SwrveConversationHelper {
```

Comment:
Good point. I'll add that comment, but I'll also move it to com.swrve.sdk.conversations.ui and make it package access only. :)